validation/ref/neuron/nrn_validation.py

```diff
@@ -129,6 +129,17 @@ def add_soma(self, diam, **kw):
         soma.gl_hh = p['gl_hh']
         soma.el_hh = p['el_hh']
 
+        # Use the following exact values calcualted using
+        # the Nernst equation, because NEURON does not use correct values
+        # for R and F.
+        soma.ena =  63.55148117386
```

Comment:
Comment should include from what concentration values these were derived (i.e. NEURON's default values in `membdef.h`) so that they can be verifiable — otherwise this is just a magic number.

Reply:
Added comment with all values used in the calculation.